src/matchmaking/output/output_fastchess.hpp

```diff
@@ -34,38 +37,57 @@ class Fastchess : public IOutput {
            << current_game_count                                  //
            << "\n";
-
-        ss << "Elo difference: "   //
-           << elo.getElo()         //
-           << ", "                 //
-           << "nElo difference: "  //
-           << elo.getnElo()        //
-           << ", "                 //
-           << "LOS: "              //
-           << Elo::getLos(stats.penta_WW, stats.penta_WD, stats.penta_WL, stats.penta_DD,
-                          stats.penta_LD, stats.penta_LL)  //
-           << ", "                                         //
-           << "PairDrawRatio: "                            //
-           << Elo::getDrawRatio(stats.penta_WW, stats.penta_WD, stats.penta_WL, stats.penta_DD,
-                                stats.penta_LD, stats.penta_LL)  //
-           << "\n";
-
+       
+        if (report_penta_ == true){
+           ss << "Elo difference: "   //
+              << eloPenta.getElo()         //
+              << ", "                 //
+              << "nElo difference: "  //
+              << eloPenta.getnElo()        //
+              << ", "                 //
+              << "LOS: "              //
+              << Elo::getLos(stats.penta_WW, stats.penta_WD, stats.penta_WL, stats.penta_DD,
+                             stats.penta_LD, stats.penta_LL)  //
+              << ", "                                         //
+              << "PairDrawRatio: "                            //
+              << Elo::getDrawRatio(stats.penta_WW, stats.penta_WD, stats.penta_WL, stats.penta_DD,
+                                   stats.penta_LD, stats.penta_LL)  //
+              << "\n";
+        } else {
+           ss << "Elo difference: "   //
+              << elo.getElo()         //
+              << ", "                 //
+              << "nElo difference: "  //
+              << elo.getnElo()        //
+              << ", "                 //
+              << "LOS: "              //
+              << Elo::getLos(stats.wins, stats.losses, stats.draws)  //
+              << ", "                                         //
+              << "DrawRatio: "                            //
+              << Elo::getDrawRatio(stats.wins, stats.losses, stats.draws)  //
+              << "\n";
+        }
         std::cout << ss.str() << std::flush;
     }
 
     void printSprt(const SPRT& sprt, const Stats& stats) override {
         if (sprt.isValid()) {
             std::stringstream ss;
 
-            ss << "LLR: " << std::fixed << std::setprecision(2)
-               << sprt.getLLR(stats.penta_WW, stats.penta_WD, stats.penta_WL, stats.penta_DD,
-                              stats.penta_LD, stats.penta_LL)
-               << " " << sprt.getBounds() << " " << sprt.getElo() << "\n";
+            if (report_penta_ == true){
+               ss << "LLR: " << std::fixed << std::setprecision(2)
+                  << sprt.getLLR(stats.penta_WW, stats.penta_WD, stats.penta_WL, stats.penta_DD,
```

Comment:
just have the getLLR instead the if branches the other stuff can be around it

Reply:
fixed with https://github.com/Disservin/fast-chess/pull/229/commits/59b00530813341486ef802b53d47efebe145cc3d